### PR TITLE
Fixed .gitignore to ignore make dist files

### PR DESCRIPTION
**Pull request by bhroam**
#### Bug/feature Description
* The .gitignore file does not ignore files generated by make dist.  pbspro-18.1.tar.gz file was checked in by accident because of this.

#### Solution Description
* Add *.tar.gz to the .gitignore file.

#### Checklist:

- [X] I have joined the **[pbspro community forum](http://community.pbspro.org/)**.
- [X] My pull request contains a **single, signed** commit. See **[setting up gpg signature](https://pbspro.atlassian.net/wiki/display/DG/Signing+Your+Git+Commits)**.
- [X] My code follows the **[coding style](https://pbspro.atlassian.net/wiki/display/DG/Coding+Standards)** of this project.
- [ ] My change requires project documentation. See **[required documentation checklist](https://pbspro.atlassian.net/wiki/display/DG/Checklist+for+Developing+Features+and+Bug+Fixes)** for details.
   - [ ] I have added documentation in the **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**.
- [ ] I have added new **PTL test(s) to my commit**. (See **[using PTL for testing](https://pbspro.atlassian.net/wiki/display/DG/Using+PTL+for+Testing)**) *(or)*
   - [ ] I have added  **manual test(s) to this pull request and explained why PTL is not appropriate** for this case.
- [ ] All new and existing automated tests have passed. (See **[running automated PTL tests](https://pbspro.atlassian.net/wiki/display/DG/PTL+Quick+Start+Guide)**).
- [ ] I have attached **test logs to this pull request** as evidence of testing/verification.

__***For further information please visit the [Developer Guide Home](https://pbspro.atlassian.net/wiki/display/DG/Developer+Guide+Home).***__

### CODE REVIEW

**Comment by agrawalravi90** [comment]
Looks good to me, thanks for the contribution.

**Comment by agrawalravi90** [comment]
@bhroam please rebase so that it can be merged.

File: .gitignore [hunk 1]
```diff
@@ -234,3 +234,5 @@ win_configure/projects.VS2008/pbs_ifl_wrap.c
 src/cmds/scripts/modulefile
 src/cmds/scripts/pbs.service
 
+# Generated by make dist
+*.tar.gz
```
Comment (mike0042) [inline .gitignore]: If you're going to ignore archive files, I suggest you add a few more...
*.tar
*.tar.bz
*.tgz
*.zip
*.cpio
*.rpm
*.deb

There may be many others, but you get the idea.

**Comment by bhroam** [comment]
Thanks for the idea @mike0042.  I updated the .gitignore file to include the ones you suggested.

**Comment by mike0042** [comment]
Looks good.

**Comment by bhroam** [comment]
Thanks for the sign offs.  I've squashed and rebased.

**Comment by agrawalravi90** [comment]
Merged it.

**Pull request merged**